lib/config.js

```diff
@@ -39,6 +39,7 @@ export const getConfig = ({reporterOptions = {}} = {}) => {
         join(templatesPath, 'matrix.hbs'),
       statistics: reporterOptions.statistics ||
         join(templatesPath, 'test-statistics.hbs'),
-    }
+    },
+    stats: Boolean(reporterOptions.stats ?? false)
```

Comment:
My brain stumbled on this.  Would the more common(?) double negation pattern to coerce into a boolean be better here?
```suggestion
    stats: !!reporterOptions.stats
```